lib/file_generator.dart

```diff
@@ -417,6 +431,39 @@ import 'package:protobuf/protobuf.dart';
     return out.toString();
   }
 
+  /// Returns the contents of the .pbgrpc.dart file for this .proto file.
+  String generateGrpcFile(
+      [OutputConfiguration config = const DefaultOutputConfiguration()]) {
+    if (!_linked) throw new StateError("not linked");
+    var out = new IndentingWriter();
```

Comment:
Maybe, or at least not care too much about the formatting here. It's only the gRPC generator output that gets formatted for now, but I do plan to format everything eventually.